### PR TITLE
Temple v2 deploy scripts

### PR DESCRIPTION
**Pull request by frontier159**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [ ] Code follows the style guide
- [ ] I have performed a self-review of my own code
- [ ] New and existing tests pass locally
- [ ] This PR is targeting the correct branch

### CODE REVIEW

File: protocol/scripts/deploys/sepolia/tlc/99-tlc-post-deployment.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            TEMPLE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.CIRCUIT_BREAKERS.TEMPLE,
+            TEMPLE_V2_ADDRESSES.CORE.CIRCUIT_BREAKER_PROXY,
+            ["preCheck"],
+            true
+        );
+    }
+
+    // Set TLC strategy
+    {   
+        await mine(TEMPLE_V2_INSTANCES.TEMPLE_LINE_OF_CREDIT.INSTANCE.setTlcStrategy(TEMPLE_V2_ADDRESSES.STRATEGIES.TLC_STRATEGY.ADDRESS));
```
Comment (frost-ostrich) [inline protocol/scripts/deploys/sepolia/tlc/99-tlc-post-deployment.ts:56]: fyi I found this issue in my PR, `setTlcStrategy` has to be after `addStrategy`

Comment (frontier159) [reply protocol/scripts/deploys/sepolia/tlc/99-tlc-post-deployment.ts:56]: Good pickup, thanks. Shows why the deployment scripts on forked mainnet are so useful!

Are you planning on fixing in your branch and will raise a PR to this branch?

Comment (frost-ostrich) [reply protocol/scripts/deploys/sepolia/tlc/99-tlc-post-deployment.ts:56]: I'm fixing my branch, I could rebase to this branch, or you could update the pr, I review and the merge to stage, whichever option you prefer :)

Comment (frost-ostrich) [reply protocol/scripts/deploys/sepolia/tlc/99-tlc-post-deployment.ts:56]: actually, thinking it better, I'll raise the PR to this branch, will be easier prob

**Review by frost-ostrich — APPROVED** [review]

**Review by mirionic — APPROVED** [review]

**Pull request merged**